yb-voyager/src/srcdb/oracle.go

```diff
@@ -572,25 +572,47 @@ func (ora *Oracle) GetPartitions(tableName sqlname.NameTuple) []string {
 	panic("not implemented")
 }
 
+var oraQueryTmplForUniqCols = `
+WITH unique_constraints AS (
+    SELECT
+        consCols.TABLE_NAME,
+        consCols.COLUMN_NAME
+    FROM
+        ALL_CONS_COLUMNS consCols
+    JOIN
+        ALL_CONSTRAINTS cons ON cons.CONSTRAINT_NAME = consCols.CONSTRAINT_NAME
+    WHERE
+        cons.CONSTRAINT_TYPE = 'U'
+        AND cons.OWNER = '%s'
+        AND consCols.TABLE_NAME IN ('%s')
+),
+unique_indexes AS (
+    SELECT
+        indCols.TABLE_NAME,
+        indCols.COLUMN_NAME
+    FROM
+        ALL_IND_COLUMNS indCols
+    JOIN
+        ALL_INDEXES ind ON ind.INDEX_NAME = indCols.INDEX_NAME
+    WHERE
+        ind.UNIQUENESS = 'UNIQUE'
```

Comment:
Will this exclude primary keys? (asking because pg required an explicit exclusion)

Reply:
yeah that needs to be added.
It might not be failing right now but that will cause false positives.

just wondering, currently we dont support changing of PK during live migration, right?
But once we start supporting that, we might need to handle it in a similar way for PK as well.

Reply:
Yeah, and false positives will cause slowness (because we will be stopping the pipeline for every event essentially). Let's fix this. 

>currently we dont support changing of PK during live migration, right?
We do. If this happens, it comes as two events: a DELETE, followed by a CREATE. But they don't really need to be executed in order because their PKs will be different.

Reply:
got it. thanks for clarifying.